common/perception_utils/include/perception_utils/perception_utils.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PERCEPTION_UTILS__PERCEPTION_UTILS_HPP_
+#define PERCEPTION_UTILS__PERCEPTION_UTILS_HPP_
+
+#include "perception_utils/geometry.hpp"
+#include "tier4_autoware_utils/geometry/boost_geometry.hpp"
+#include "tier4_autoware_utils/geometry/boost_polygon_utils.hpp"
+
+#include <boost/geometry.hpp>
+
+#include <algorithm>
+#include <vector>
+
+namespace perception_utils
+{
+std::uint8_t getHighestProbLabel(
+  const std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> & classification);
+
+autoware_auto_perception_msgs::msg::TrackedObject toTrackedObject(
+  const autoware_auto_perception_msgs::msg::DetectedObject & detected_object);
+
```

Comment:
If you are ok, can you add the following functions?
- [toDetectedObject](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/detection_by_tracker/src/detection_by_tracker_core.cpp#L102-L121)
- transformObjects
  - https://github.com/autowarefoundation/autoware.universe/blob/main/perception/multi_object_tracker/src/multi_object_tracker_core.cpp#L58
  - https://github.com/autowarefoundation/autoware.universe/blob/main/perception/detection_by_tracker/src/detection_by_tracker_core.cpp#L48

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1377/commits/97b68835e75f0c49eb95a4b162357583b0b1d1b0
DONE

Reply:
Perfect :+1: